x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/search_or_filter.cy.ts

```diff
@@ -53,18 +55,17 @@ describe('Timeline search and filters', { tags: ['@ess', '@serverless'] }, () =>
     });
   });
 
-  // FLAKY: https://github.com/elastic/kibana/issues/169882
-  describe.skip('Update kqlMode for timeline', () => {
+  describe('Update kqlMode for timeline', () => {
     beforeEach(() => {
       login();
+      deleteTimelines();
       visit(TIMELINES_URL);
       waitForTimelinesPanelToBeLoaded();
       openTimelineUsingToggle();
       cy.intercept('PATCH', '/api/timeline').as('update');
       cy.get(LOADING_INDICATOR).should('not.exist');
       showDataProviderQueryBuilder();
       cy.get(TIMELINE_SEARCH_OR_FILTER).click();
-      cy.get(TIMELINE_SEARCH_OR_FILTER).should('exist');
     });
 
     it('should be able to update timeline kqlMode with filter', () => {
```

Comment:
the line below (`cy.get(TIMELINE_KQLMODE_FILTER).click();`) (and also line 82 `cy.get(TIMELINE_KQLMODE_SEARCH).click();`) we should extract the click actions into tasks

Reply:
Done https://github.com/elastic/kibana/pull/173367/commits/76a9814698b1d07820f94193cb3250d818ef2360